lib/IMAP/FolderMapper.php

```diff
@@ -142,10 +150,17 @@ public function getFoldersStatusAsObject(Horde_Imap_Client_Socket $client,
 
 		$statuses = [];
 		foreach ($multiStatus as $mailbox => $status) {
-			$statuses[$mailbox] = new MailboxStats(
-				$status['messages'],
-				$status['unseen'],
-			);
+			try {
+				if(!isset($status['messages']) && !isset($status['unseen'])) {
```

Comment:
```suggestion
				if (!isset($status['messages'], $status['unseen'])) {
```

coding style and simplification

https://www.php.net/manual/en/function.isset.php